includes/internal/emails/patterns/student-no-progress-7-days.php

```diff
@@ -0,0 +1,48 @@
+
+	<!-- wp:group {"style":{"color":{"background":"#f6f7f7"},"spacing":{"padding":{"top":"32px","right":"32px","bottom":"32px","left":"32px"},"margin":{"top":"48px"}}}} -->
+	<div class="wp-block-group has-background" style="background-color:#f6f7f7;margin-top:48px;padding-top:32px;padding-right:32px;padding-bottom:32px;padding-left:32px"><!-- wp:paragraph {"style":{"typography":{"fontSize":"32px"},"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"margin":{"top":"0px","right":"0px","bottom":"24px","left":"0px"}},"color":{"text":"#101517"}}} -->
+		<p class="has-text-color" style="color:#101517;font-size:32px;margin-top:0px;margin-right:0px;margin-bottom:24px;margin-left:0px;padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><strong>[student:displayname]</strong></p>
+		<!-- /wp:paragraph -->
+
+		<!-- wp:paragraph {"style":{"typography":{"fontSize":"16px"},"spacing":{"margin":{"top":"0px","bottom":"0px"}},"color":{"text":"#101517"}}} -->
+		<p class="has-text-color" style="color:#101517;font-size:16px;margin-top:0px;margin-bottom:0px"><strong>Course Name</strong></p>
+		<!-- /wp:paragraph -->
+
+		<!-- wp:paragraph {"style":{"typography":{"fontSize":"16px"},"color":{"text":"#101517"},"spacing":{"margin":{"top":"0","bottom":"0"}}}} -->
+		<p class="has-text-color" style="color:#101517;font-size:16px;margin-top:0;margin-bottom:0">[course:name]</p>
+		<!-- /wp:paragraph -->
+
+		<!-- wp:paragraph {"style":{"typography":{"fontSize":"16px"},"spacing":{"margin":{"top":"24px","bottom":"0px"}},"color":{"text":"#101517"}}} -->
+		<p class="has-text-color" style="color:#101517;font-size:16px;margin-top:24px;margin-bottom:0px"><strong>Next Lesson</strong></p>
+		<!-- /wp:paragraph -->
+
+		<!-- wp:paragraph {"style":{"typography":{"fontSize":"16px"},"color":{"text":"#101517"},"spacing":{"margin":{"top":"0","bottom":"0"}}}} -->
+		<p class="has-text-color" style="color:#101517;font-size:16px;margin-top:0;margin-bottom:0">[lesson:name]</p>
+		<!-- /wp:paragraph -->
+
+		<!-- wp:buttons {"style":{"spacing":{"margin":{"top":"40px"}}}} -->
+		<div class="wp-block-buttons" style="margin-top:40px"><!-- wp:button {"style":{"spacing":{"padding":{"top":"16px","bottom":"16px","left":"20px","right":"20px"}},"color":{"background":"#101517","text":"#fefefe"},"typography":{"fontSize":"16px"},"border":{"radius":"4px"}}} -->
+			<div class="wp-block-button has-custom-font-size" style="font-size:16px"><a class="wp-block-button__link has-text-color has-background" href="[resume:url]" style="border-radius:4px;background-color:#101517;color:#fefefe;padding-top:16px;padding-right:20px;padding-bottom:16px;padding-left:20px">Resume Course</a></div>
+			<!-- /wp:button --></div>
+		<!-- /wp:buttons -->
+
+		<!-- wp:paragraph {"style":{"typography":{"lineHeight":"1"},"spacing":{"padding":{"top":"24px"}}}} -->
```

Comment:
Just a small suggestion, more like an opinion, not too important, if this top part is common among all the patterns, can we move them to another file and require that file inside these templates?